### ISSUE TITLE
Simple table billing mode

### ISSUE THREAD

**Issue by dave-graham** [awslabs/serverless-application-model#696]
Hi,

Given that dynamo now has a much simpler method to auto scale, and this fits with serverless (autoscaling as demand increases, and only in-curing cost for data stored), can the default for simple table be PAY_PER_REQUEST billing mode, and the requirement for provisioned throughput be made optional?

I'm happy to do this using AWS::DynamoDB::Table for now, but feels like a much simpler default, and much more closely aligned to how lambda works.

**Comment by jlhood** [comment]
Great suggestion! The fastest way to get this in would be to submit a PR. Here are the relevant files to update for the functional change. Then you would need to add relevant unit tests and update the documentation spec.

https://github.com/awslabs/serverless-application-model/blob/develop/samtranslator/model/sam_resources.py#L478

https://github.com/awslabs/serverless-application-model/blob/develop/samtranslator/model/dynamodb.py#L6

**Comment by Jacco** [comment]
Am working on a PR for this, but also saw @dave-graham forked this repo on 6-dec so maybe he is working on it too. Shall I create the PR, or do you want to do it @dave-graham?

**Comment by metaskills** [comment]
I saw that #705 said this was merged, but was it reverted?

**Comment by keetonian** [comment]
It was merged into develop and is waiting on our next release to be merged into master.

**Comment by keetonian** [comment]
Released!
